e2e/aws/main_test.go

```diff
@@ -74,6 +74,23 @@ const (
 	// rssDBUserEnv is the name of the IAM role that tests will use as a
 	// database user to connect to Redshift Serverless.
 	rssDBUserEnv = "REDSHIFT_SERVERLESS_IAM_DB_USER"
+	// redshiftAccessRoleEnv is the environment variable that specifies the IAM
+	// role that Teleport Database Service will assume to access Redshift
+	// cluster databases.
+	// See modules/databases-ci/ from cloud-terraform repo for more details.
+	redshiftAccessRoleEnv = "REDSHIFT_ACCESS_ROLE"
+	// redshiftDiscoveryRoleEnv is the environment variable that specifies the
+	// IAM role that Teleport Discovery Service will assume to discover
+	// Redshift cluster databases.
+	// See modules/databases-ci/ from cloud-terraform repo for more details.
+	redshiftDiscoveryRoleEnv = "REDSHIFT_DISCOVERY_ROLE"
+	// redshiftNameEnv is the environment variable that specifies the
+	// name of the Redshift cluster db that will be created by the
+	// Teleport Discovery Service.
+	redshiftNameEnv = "REDSHIFT_CLUSTER_NAME"
+	// redshiftDBUserEnv is the name of the IAM role that tests will use as a
+	// database user to connect to Redshift Serverless.
+	redshiftDBUserEnv = "REDSHIFT_IAM_DB_USER"
```

Comment:
nit: for clarity, maybe
```suggestion
	redshiftIAMDBUserEnv = "REDSHIFT_IAM_DB_USER"
```